sdk/resourcemanager/azure-resourcemanager-redis/src/main/java/com/azure/resourcemanager/redis/implementation/RedisCacheImpl.java

```diff
@@ -586,6 +587,9 @@ public Mono<RedisCache> updateResourceAsync() {
     public Mono<RedisCache> createResourceAsync() {
         createParameters.withLocation(this.regionName());
         createParameters.withTags(this.innerModel().tags());
+        if (CoreUtils.isNullOrEmpty(this.createParameters.redisVersion())) {
+            withRedisVersion(RedisVersion.V6);
```

Comment:
which version does RedisVersion.V6 refer? 6.0.0?

Reply:
It's major redis version(e.g. 4, 6), not a specific(minor, e.g. 6.0.0) one. Service only allow configuring with major version, and in backend, they will choose a minor version(I assume the latest).